src/fontra/backends/designspace.py

```diff
@@ -682,14 +680,17 @@ def buildUFOLayerGlyph(
         if component.location:
             # It's a variable component
             varCoDict = {"base": component.name, "location": component.location}
-            if component.transformation != Transformation():
+            if isinstance(component.transformation, DecomposedTransform):
```

Comment:
I would like your attention in this line @justvanrossum. I am not sure if I understand it correctly

Reply:
No, the original check was correct, and should be replaced with 
```python
    if component.transformation != DecomposedTransform():
```
It compares `component.transformation` with the *identity transform* (ie. "no transformation")

Reply:
Should I create an DecomposedTransformIdentity? There's misc.transform.Transform.Identity for regular ones.

Reply:
I might be a small optimization, but let's not do that. It's also conceptually not quite right: `Transform` instances are immutable, but `DecomposedTransform` instances are not, so there's no guarantee that `DecomposedIdentity` remains the identity (even though no code would be supposed to change it).